Formula/tarantool.rb

```diff
@@ -14,8 +14,11 @@ class Tarantool < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "autoconf"
+  depends_on "automake"
   depends_on "icu4c"
-  depends_on "openssl@1.1"
+  depends_on "libtool"
+  depends_on "openssl"
```

Comment:
OpenSSL 1.0 is going EOL on December 31, so we want to stick with `openssl@1.1`

Reply:
Ok, I'll rollback the change.

> Regarding downgrading version: how unstable is the 2.2 “beta” version, and when is a 2.2 stable going to be released? It's unfortunate that we shipped a beta version, but going back is also not optimal… trying to figure out what the best solution is here.

That is not too bad to leave the default version as it is. But we need to publish additionally LTS and stable versions. May be we can publish it in the following way?
- Formula/tarantool.rb -> default version not changing and points to the current 2.2.1
- Formula/tarantool@2.1.rb -> new additional version points to the stable 2.1.2
- Formula/tarantool@1.10.rb -> new additional version points to the LTS 1.10.4